wasm.md

```diff
@@ -39,6 +51,8 @@ Integers in K are unbounded.
 As an invariant, however, for any integer `< iNN > I:Int` on the stack, `I` is between 0 and `#pow(NN) - 1`.
 That way, unsigned instructions can make use of `I` directly, whereas signed instructions may need `#signed(iNN, I)`.
 
+The highest address in a memory instance divided by the `#pageSize` may not exceed the value in `<max>`, if present.
```

Comment:
Let's say "in the `<max>` cell", so that it's clear we're talking about the K cell `<max>` and not some meta-level parameter. Also `#pageSize => #pageSize()`, and say "the `#pageSize()` constant (defined below)".

Reply:
Done